### ISSUE TITLE
Allow `use` and `select` as a descendant of `amp-nested-menu`

### ISSUE THREAD

**Issue by functionfirst** [ampproject/amphtml#29300]
## Describe the new feature or change to an existing feature you'd like to see
I'm currently implementing an amp-nested-menu into a sidebar where I'm receiving amp validation errors for using SVG's with the `<use>` tag as well as similar errors using a `<select>` tag.

## Describe alternatives you've considered

We could use an `<amp-img>` referencing the svg as a src rather than use inline svg tags. However, this would require some significant refactoring of our current AMP platform to serve these svg's.

Alternatives for `<select>`. While we could implement this as a standard html list we have deliberately used a `<select>` tag so our users can use their devices with native select UI instead of a custom html selector.

## Additional context

Displaying a right angle arrow alongside some navigation text shown in the screenshot below:

![image](https://user-images.githubusercontent.com/784829/87674148-83833c00-c76d-11ea-9a6e-11887f77a201.png)

The paired back mark-up looks like this:

```
<h4 amp-nested-submenu-open="">
  Accessories 
  <svg width="15" height="15" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
    <use height="15" width="15" href="#svg_angle-right" xlink:href="#svg_angle-right"></use>
  </svg>
</h4>
```

As you can see, rather than rendering the svg content directly we re-use the same svg multiple times using the <use> tag.

These tags seem to be missing from [validator-amp-nested-menu.protoascii](https://github.com/catherinehoule/amphtml/blob/bd6ab3dfc5b07de85e2e9017ad791d9291658091/extensions/amp-nested-menu/validator-amp-nested-menu.protoascii), presumably they just need to be added to the list?
